aws-ecr-repository/aws-ecr-repository.json

```diff
@@ -13,7 +13,8 @@
                 "RegistryId": {
                     "$ref": "#/definitions/RegistryId"
                 }
-            }
+            },
+            "additionalProperties": false
```

Comment:
Will this block existing policy with additional properties?

Reply:
Yes. This is intentional to prevent templates with incorrect inputs.